VideoLocker/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -223,6 +224,18 @@ public ProfileModel getProfile() throws Exception {
         // hold the json string as it is
         res.json = json;
 
+
+        // store profile json
+        if (json != null ) {
```

Comment:
@aleffert  I am not sure about current approach. 
If we check the Crashlytics logs, there are some none-fatal failure in parsing json data returned from server. I agree we should check the status code.  we will address this issue in migrating code to retrofit framework